embedded-files/fuseml-core-deployment.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fuseml-core
+  namespace: fuseml-core
+  labels:
+    app.kubernetes.io/name: fuseml-core
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: fuseml-core
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: fuseml-core
+    spec:
+      containers:
+      - name: fuseml-core
+        image: ghcr.io/fuseml/fuseml-core:latest
+        env:
+          - name: GITEA_USERNAME
+            valueFrom:
+              secretKeyRef:
+                key: GITEA_USERNAME
+                name: fuseml-core-gitea
+          - name: GITEA_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: GITEA_PASSWORD
+                name: fuseml-core-gitea
+          - name: GITEA_URL
+            valueFrom:
```

Comment:
Mentioned before: the URL could be supplied directly instead of loading it from a secret.
... Unless using this file as a template is too time consuming. I realize all other foundation services are deployed using helm charts, while this here is just a standalone deployment, so what I'm suggesting doesn't fit the pattern. If that's the case, please ignore my suggestion.

Reply:
Exactly, the reason I did it was that I was already creating secret, so I did not need to modify the deployment template. But I guess I can do it just for clarity.